foundation/foundation-mda/src/main/java/com/boozallen/aiops/mda/metamodel/element/python/PythonStep.java

```diff
@@ -135,8 +136,9 @@ private Set<String> getImportsWithRelativePaths(boolean isImplModule) {
         for (String moduleImport : imports) {
             if (moduleImport.startsWith("from record.")
                     || moduleImport.startsWith("from dictionary.")) {
+                String rootArtifact = PythonGeneratorUtils.rootArtifactId.replace("-", "_");
                 importsWithRelativePaths.add(moduleImport.replace("from ",
```

Comment:
A: The name of this method/variable seems misleading now. These aren't relative imports.

Reply:
yup now it isrelative imports anymore, Ill update to something like getDataRecordsAndDictImports.